src/broker/content-cache.c

```diff
@@ -774,6 +774,8 @@ static void content_flush_request (flux_t *h, flux_msg_handler_t *mh,
     struct content_cache *cache = arg;
 
     if (cache->acct_dirty > 0) {
+        if (cache_flush (cache) < 0)
```

Comment:
Commit message typo: ..."when a cache flush has not yet initiated" -- "been initiated"?